internal/contrib/telemetrytest/telemetry_test.go

```diff
@@ -35,7 +35,8 @@ func (p *contribPkg) hasInstrumentationImport() bool {
 
 // TestTelemetryEnabled verifies that the expected contrib packages leverage instrumentation telemetry
 func TestTelemetryEnabled(t *testing.T) {
-	root, err := filepath.Abs("../../../contrib")
+	p := fmt.Sprintf("..%c..%c..%ccontrib", os.PathSeparator, os.PathSeparator, os.PathSeparator)
+	root, err := filepath.Abs(p)
```

Comment:
I think you can use `filepath.Join` for this: https://pkg.go.dev/path/filepath#Join